http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -1728,6 +1728,45 @@ paths:
                 items:
                   $ref: "#/components/schemas/ListAppResponse"
 
+  "/api/apps/v2/{workspaceId}/deleteAll":
+    post:
+      summary: Deletes ALL apps associated with the given workspace Id
+      description: deletes all Apps in a workspace
+      operationId: deleteAllAppsV2
+      tags:
+        - appsV2
+      parameters:
+        - in: path
+          name: workspaceId
+          description: workspaceId
+          required: true
+          schema:
+            type: string
+        - in: query
+          name: deleteDisk
+          description: Whether or not the disk associated with the apps should be deleted. Default to false if not specified.
+          required: false
+          schema:
+            type: boolean
+            default: false
+      responses:
+        "202":
+          description: App(s) deletion request accepted
```

Comment:
So this call is asynchronous, correct? That means that we will not have a way of knowing when/if the deletion ultimately succeeds. What is the suggestion for us to determine that? Should we poll WSM to see if resources still exist, or poll a getApp method? It would be great if there was a way we could poll the deletion itself (like a job control ID being returned) so that if it fails, we can present the user with the cause of failure.

Reply:
Right now leo does not support 'jobs' like WSM does. The resources themselves are only ever soft-deleted, so you can query the list endpoint with the `workspaceId` and `includeDelete=true`. Each app deletion will be processed separately and in parallel, so any apps that fail deletion will be marked with `Error`, and any error(s) included in the list response.